docs/manuals/admin/crate.md

```diff
@@ -24,4 +24,21 @@ it is using its admin interface, as documented [here](https://crate.io/docs/clie
 Alternatively, you can use its [HTTP api](https://crate.io/docs/crate/getting-started/en/latest/first-use/query.html#the-cratedb-http-endpoint),
 or any of its [supported clients](https://crate.io/docs/crate/tutorials/en/latest/getting-started/start-building/index.html).
 
+**Note**- Crate(3.x and 4.x) does not support nested arrays.
```

Comment:
i think this is better in troubleshooting docs

Reply:
> i think this is better in troubleshooting docs

@chicco785 , I have added description in troubleshooting docs.